bin/citrea/src/main.rs

```diff
@@ -201,6 +205,25 @@ where
         None => FullNodeConfig::from_env()
             .context("Failed to read rollup configuration from the environment")?,
     };
+
+    let telemetry_addr: SocketAddr = format!(
+        "{}:{}",
+        rollup_config.telemetry.bind_host, rollup_config.telemetry.bind_port
+    )
+    .parse()
+    .map_err(|_| anyhow!("Invalid telemetry address"))?;
+
+    debug!("Starting telemetry server on: {}", telemetry_addr);
+
+    let builder = PrometheusBuilder::new().with_http_listener(telemetry_addr);
+    builder
+        .idle_timeout(
+            MetricKindMask::GAUGE | MetricKindMask::HISTOGRAM,
```

Comment:
not everyone's going to run telemetry. let's make it optional in the config. if telemetry config is given, we start the server, if not, we don't.